### PR TITLE
fix: proper logs location

### PR DESCRIPTION
**Pull request by TCattd**
Fixes https://github.com/TransbankDevelopers/transbank-plugin-woocommerce-webpay-rest/issues/118

It will create a new folder transbank_webpay_plus_rest inside the uploads directory, and then will use a logs folder inside it. So logs final path will be: uploads/transbank_webpay_plus_rest/logs/

This way logs will not be lost if the plugin gets upgraded to a new version by any means.

BTW, i think those two defines (Transbank_webpay_Rest_Webpay_ROOT and the new one for the uploads folder) should be inside the main plugin file, webpay-rest.php, to be used in every file if they are needed. As the usual plugins behavior on WP.

Esteban

### CODE REVIEW

File: plugin/src/Helpers/LogHandler.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 }
 
 define('Transbank_webpay_Rest_Webpay_ROOT', dirname(dirname(__DIR__)));
```
Comment (FongWan) [inline plugin/src/Helpers/LogHandler.php:14]: This constant will no longer be used after the changes, it must be removed.

**Review by FongWan — CHANGES_REQUESTED** [review]
This PR will effectively make the logs permanent despite plugins updates, but the unused constant should be removed before merge.

As an additional note, because transbank log file names are not hashed, there is a possibility of leakage of sensitive data by accessing it directly with the browser (this is regardless of whether it is in the `/plugins/` folder or `/uploads/` folder), so we need another commit to fix this problem.

**Comment by TCattd** [comment]
Taken care of both things, @FongWan
Thanks for the heads up.

Esteban

**Review by FongWan — APPROVED** [review]

**Comment by mastudillot** [comment]
Hicimos cambios en el plugin y vamos a tomar los cambios propuestos y los llevaremos a la nueva versión, muchas gracias por la colaboración.

**Comment by TCattd** [comment]
Super @mastudillot 

Mientras se resuelva este problema, que ahora existe en transbank-plugin-woocommerce-webpay-rest/plugin/src/Helpers/TbkFactory.php, todo bien para los clientes :)

Gracias!

**Pull request closed without merging**